.snippets/code/develop/parachain-devs/runtime-development/FRAME/pallet-testing/event-testing.rs

```diff
@@ -0,0 +1,18 @@
+#[test]
```

Comment:
I think this is not the actual snippet right? Because there is no event testing in here. It is the same snippet as function-call-testing